master/manifests/site.pp

```diff
@@ -76,31 +76,16 @@
   notify => Service['jenkins-slave'],
 }
 
-package { 'git':
-  ensure => 'installed',
-}
-
-package { 'subversion':
-  ensure => 'installed',
-}
 
 ### Jenkins Plugins
 
-jenkins::plugin {
-  'bazaar': ;
-}
-
-package { 'bzr':
-  ensure => 'installed',
-}
-
-jenkins::plugin {
-  'build-timeout': ;
-}
-
-jenkins::plugin {
-  'collapsing-console-sections': ;
-}
+# required by groovy-postbuild
+jenkins::plugin { 'ace-editor': ; }
```

Comment:
Where did you find this? I don't see it anywhere in here: https://github.com/jenkinsci/groovy-postbuild-plugin/blob/master/pom.xml